docs/evaluation/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -13,70 +13,96 @@ Before diving into this content, it might be helpful to read the following:
+   of available transformations, see [our reference](/reference/data_formats/dataset_transformations)
 
-If you are creating a key-value dataset, you may optionally define a schema for your dataset. All examples you create will be validated against this schema.
+* **Create from template**: If you plan to collect production traces in your dataset from LangChain 
+[ChatModels](https://python.langchain.com/api_reference/core/language_models/langchain_core.language_models.chat_models.BaseChatModel.html) 
+or from OpenAI calls using the [LangSmith OpenAI wrapper](/observability/how_to_guides/tracing/annotate_code#wrap-the-openai-client), we offer a prebuilt schema and transformation that makes it easy to
+collect these runs in format that is easy to use downstream. This is entirely customizable, and does not lock your dataset into a set format. 
 
-![Defining dataset schema](./static/dataset_schema_definition.png)
+   Please see the [dataset transformations reference](/reference/data_formats/dataset_transformations) for more information. 
+   
+   :::note
+   Users who keep their prompt in their system message often prefer to add the `Remove System Messages` transformation to 
+   their dataset in addition to the default ones added by the template. This prevents the Sytem prompt, which is often 
+   what people want to evaluate, from entering the dataset. 
 
-Dataset schemas are defined with standard [JSON schemas](https://json-schema.org/). If you would rather manually enter raw JSON, click "Editor" at the bottom of the schema editor and then select "JSON".
 
-![Custom JSON schema input](./static/custom_json_schema.png)
+## Add runs to your dataset in the UI
 
-If you have defined a schema for your dataset, you will get easy validation when creating new examples:
-
-![Schema validation on example creation](./static/schema_validation.png)
-
-## Add inputs and outputs from traces to datasets
+There are a few ways to add examples to your dataset in UI. 
 
+### Add run(s) from the tracing project UI
 We typically construct datasets over time by collecting representative examples from debugging or other runs. To do this, we first filter the traces to find the ones we want to add to the dataset. Then we add the inputs and outputs from these traces to the dataset.
 
-You can do this from any 'run' details page by clicking the 'Add to Dataset' button in the top right-hand corner.
-
 :::tip
 An extremely powerful technique to build datasets is to drill-down into the most interesting traces, such as traces that were tagged with poor user feedback, and add them to a dataset.
 For tips on how to filter traces, see the [filtering traces](../../../observability/how_to_guides/monitoring/filter_traces_in_application) guide.
 :::
 
-:::tip automations
+There are two ways to add data from tracing projects to datasets. When runs are added from tracing projects to 
```

Comment:
these two sentences dont feel like they should be next to each other. maybe second should be in the first para?